omod/src/main/resources/config.xml

```diff
@@ -11,10 +11,10 @@
 		See <a href="http://openmrs.org/wiki/Idgen_Module">http://openmrs.org/wiki/Idgen_Module</a> for more information
 	</description>
 	<activator>@MODULE_PACKAGE@.IdgenModuleActivator</activator>
-	<require_version>1.9.2</require_version>
 	<require_modules>
 		<require_module>org.openmrs.module.webservices.rest</require_module>
 	</require_modules>
+	<require_version>1.11.3, 1.10.2 - 1.10.*, 1.9.9 - 1.9.*</require_version>
```

Comment:
Curious about this?  Do we need those upper bounds or are they implied?  And if we need them, don't we also need them on the 1.11 line and above?